### PR TITLE
Add Store Loading State

### PR DESCRIPTION
**Pull request by pivilartisant**
https://github.com/massalabs/station/assets/90157528/f0b6c394-5336-436d-ac9d-fb4af5a21de0

### CODE REVIEW

File: web/massastation/src/pages/Store/PluginCardLoading.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import { Plugin } from '@massalabs/react-ui-kit';
+export function PluginCardLoading() {
+  const argsStore = {
+    preIcon: <img src="" alt="plugin-logo" />,
+    topAction: <FiDownload />,
+    title: 'Lorem Ipsum Plugin',
+    subtitle: 'Lorem Author',
+    subtitleIcon: null,
+    content: 'Lorem Ipsum Doloris Sit Amet',
+  };
```
Comment (Pierregvx) [inline web/massastation/src/pages/Store/PluginCardLoading.tsx:11]: Do we need this ? it seem to me that we didn't use this in the other loader, tell me if I am wrong

Comment (pivilartisant) [reply web/massastation/src/pages/Store/PluginCardLoading.tsx:11]: That's a good point I guess it can be removed

Comment (Thykof) [reply web/massastation/src/pages/Store/PluginCardLoading.tsx:11]: now it's not the same result as in you video, is it intended?

Comment (pivilartisant) [reply web/massastation/src/pages/Store/PluginCardLoading.tsx:11]: indeed it's a different results : 
[Screencast from 17-07-2023 11:04:17.webm](https://github.com/massalabs/station/assets/90157528/0ddd3e62-611a-4914-8969-ae33166c0bc8)

I would say it a product vision choice @SlnPons  between this one and the original one wdyt ?

Comment (SlnPons) [reply web/massastation/src/pages/Store/PluginCardLoading.tsx:11]: @pivilartisant I don't see the 2 options here. What is the original vs. "this one". 
Knowing that I haven't personally prioritise this task... I need more details to validate.

Comment (pivilartisant) [reply web/massastation/src/pages/Store/PluginCardLoading.tsx:11]: > @pivilartisant I don't see the 2 options here. What is the original vs. "this one". Knowing that I haven't personally prioritise this task... I need more details to validate.

The original one is in the pr description, the second one is linked in my last comment

Comment (SlnPons) [reply web/massastation/src/pages/Store/PluginCardLoading.tsx:11]: I'm sorry @pivilartisant I don't see the difference :/ 
From a product perspective, the most important is to be consistent. So the loading state MUST be the same across all our products. I do believe that the "original" loading state is already used on other pages, if that's the case, we need to use it everywhere.
The loading states on Massa Station have been implemented without my validation so i'm not across the logic here... I can't help more that by saying: it must be the same across all massa station app.

Comment (pivilartisant) [reply web/massastation/src/pages/Store/PluginCardLoading.tsx:11]: Hello @SlnPons I've checked with @LucasF243, He green lights we for the second option which is also standard across massa station & wallet :)

**Review by Thykof — APPROVED** [review]

**Pull request merged**